lib/my_inject_array.rb

```diff
@@ -0,0 +1,14 @@
+class Array
+  def my_inject(start = nil, &code)
```

Comment:
Unused method argument - `code`. If it's necessary, use `_` or `_code` as an argument name to indicate that it won't be used.